editor/import/resource_importer_scene.cpp

```diff
@@ -2369,9 +2369,8 @@ Node *ResourceImporterScene::pre_import(const String &p_source_file, const HashM
 	ERR_FAIL_COND_V(!importer.is_valid(), nullptr);
 
 	Error err = OK;
-	HashMap<StringName, Variant> options_dupe = p_options;
```

Comment:
There was `options_dupe["gltf/embedded_image_handling"] = 3;` line after it at some point. So it is redundant.